src/Test/Perf/helloworld/HelloWorld.csx

```diff
@@ -0,0 +1,20 @@
+#load "../util.csx"
```

Comment:
I believe that's what Vlad means, yes.  The idea is that the folder is effectively the "test name" and when you got to a test folder, you always run the same command to run the test.